Makefile

```diff
@@ -467,12 +467,12 @@ update-roles:
 .PHONY: bundle
 bundle: generate manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	if [ -z "$(platform)" ]; then
-		echo "[ERROR] You must specify 'platform' macros. For example: `make bundle platform=kubernetes`"
+		echo "[ERROR] You must specify 'platform' macros. For example: `make bundle platform=openshift`"
```

Comment:
Sorry, why do we need platform argument if we will support only on platform Openshfit ?

Reply:
fixed